app/assets/stylesheets/mortgage_calculator/components/layouts/_sticky.scss

```diff
@@ -0,0 +1,85 @@
+.l-stuck {
+  @include column(12);
+
+  @include respond-to($mq-m) {
+    @include column(8);
+
+    margin: 0;
+  }
+
+  max-width: 600px;
+  position: fixed;
+  margin: 0;
+  top: 0;
+  padding: $baseline-unit;
+  background-color: rgb(255, 255, 255);
+  background-color: rgba(255, 255, 255, 0.95);
+  border-bottom: 1px solid #ccc;
+  z-index: 3;
+
+  .slider--with-follower-top {
+    transition: margin 0.3s ease-out;
+    margin-top: $baseline-unit*7;
+
+    .ui-slider-handle {
+      &:after {
+        top: -$baseline-unit*1.5;
+        margin-left: -5px;
+        border-left-width: 5px;
+        border-right-width: 5px;
+        border-bottom-width: 5px;
+      }
+    }
+  }
+
+  .label-follower.label-follower--top {
```

Comment:
I suppose it would make sense if there were multiple modifiers:
```
.label-follower.label-follower--top {}
.label-follower.label-follower--bottom {}
.label-follower.label-follower--middle {}
```
But there arent, so I'm happy with this